api/users.go

```diff
@@ -127,3 +127,39 @@ func (api *API) ChangePasswordHandler(ctx context.Context, w http.ResponseWriter
 
 	return models.NewSuccessResponse(jsonResponse, http.StatusAccepted, headers), nil
 }
+
+//PasswordResetHandler requests a password reset email be sent to the user and returns a http handler interface
+func (api *API) PasswordResetHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+	defer req.Body.Close()
+
+	body, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		return nil, handleBodyReadError(ctx, err)
```

Comment:
We might need to log the event when an error is returned unless we have decided not to...

Reply:
When a new error constructors write a log entry for the error when the error is built.
See here https://github.com/ONSdigital/dp-identity-api/blob/develop/models/errors.go#L23